meeteval/viz/__main__.py

```diff
@@ -335,11 +290,38 @@ def add_argument(self, command_parser, name, p):
                     default=None,
                     nargs='+',
                 )
+            elif name == 'file_format':
+                command_parser.add_argument(
+                    '--file-format',
+                    help='The format of the file paths in the index.html file. '
+                         'The following placeholders are available: '
+                         '{system_name}, {alignment}, {session_id}.\n'
+                         'The default splits by system name and alignment type to make it'
+                         'easy to create comparisons between different systems and alignments. '
+                         'using "meeteval-viz index_html".',
```

Comment:
Is there no default? To me, it is not clear, what this argument does. The visual name in the html?